10.9-libcxx/dpkg-base-files.info

```diff
@@ -2,14 +2,37 @@ Info4: <<
 Package: dpkg-base-files
 Version: 0.4
 Revision: 1
-Depends: dpkg (>= 1.10.21-1229)
+BuildDepends: fink (>= 0.45.99.git)
+Depends: dpkg (>> 1.16)
 Source: mirror:custom:fink/%n-%v.tar.gz
```

Comment:
This is the first change to dpkg-base-files in the branch, so it needs version-up, since it's a change (can't keep same %v-%r) and it's a really really new thing not just a tweak.

Reply:
Pretty much everything there will need a rev up on launch, but I don't have it since I don't have the fink git version yet, it's on the todo list for the PR though